src/Castle.Windsor/Core/Internal/SimpleThreadSafeDictionary.cs

```diff
@@ -17,7 +17,7 @@ namespace Castle.Core.Internal
 	using System;
 	using System.Collections.Generic;
 	using System.Linq;
-
+	
```

Comment:
Indentation.

Reply:
Please "View File". It looks correct, i think this is a git diff view issues

![image](https://user-images.githubusercontent.com/2411563/101076570-dede8b00-35a3-11eb-97cf-3d683aba7e56.png)

Reply:
If I view file, I can see the extra tab you've added like shown in the diff.